src/components/macros/playSound.js

```diff
@@ -97,6 +97,14 @@ function PlaySound({updateArgs, args, stations, clients, playSound}) {
           value={sound.channel}
           onChange={e => updateSound("channel", e.target.value.split(","))}
         />
+        <div style={{ display: "flex", gap: '1rem', padding: '20px' }}>
+          <Label>Preserve Audio Channels <small>Advanced</small> </Label>
```

Comment:
I would change this to "Preserve Surround Channels <small>Advanced</small>" to make it a little bit clear what its purpose is.

Reply:
How would you feel about adding a popover with some help text about what these options (Channels and Preserve Surround Channels) do?